platform-api/src/services/cf_client.py

```
@@ -0,0 +1,115 @@
+          ) {
+            dimensions {
+              datetimeHour
+            }
+            sum {
+              double1
+            }
+          }
+        }
+      }
+    }
+    """ % (account_id, datetime_geq, metric)
```

Ruff (0.15.11)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)